dubbo-config/dubbo-config-spring/src/main/resources/META-INF/dubbo.xsd

```diff
@@ -6,7 +6,7 @@
             targetNamespace="http://dubbo.apache.org/schema/dubbo">
 
     <xsd:import namespace="http://www.w3.org/XML/1998/namespace"/>
-    <xsd:import namespace="http://www.springframework.org/schema/beans"/>
+    <xsd:import namespace="http://www.springframework.org/schema/beans" schemaLocation="http://www.springframework.org/schema/beans/spring-beans-4.3.xsd"/>

```

Comment:
Please change `http://www.springframework.org/schema/beans/spring-beans-4.3.xsd` to `http://www.springframework.org/schema/beans/spring-beans.xsd`